src/main/java/org/veupathdb/service/eda/ds/plugin/pass/MapPlugin.java

```diff
@@ -172,12 +156,67 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
         .put("maxLat", data.maxLat)
         .put("maxLon", data.maxLon)
         .toString()
-        .getBytes()
+        .getBytes(StandardCharsets.UTF_8)
       );
     }
     // add config and close
     String config = "],\"config\":{\"completeCasesGeoVar\":" + entityRecordsWithGeoVar + "}}";
     out.write(config.getBytes());
     out.flush();
   }
+
+  private static class ParsedNumericViewport {
```

Comment:
Wondering if you need this extra class.  Could just store off four private double fields: minLat, maxLat, minLon, maxLon, plus containsDateLine boolean.